conf/application.conf

```diff
@@ -105,11 +105,11 @@ microservice {
   services {
     govuk-notify {
       api-key {
-        iss-uuid = no-default-credential
-        secret-key-uuid = no-default-credential
+        iss-uuid = add5f813-2bbc-4855-a88b-3214ed8ad7bb
+        secret-key-uuid = 805c68c6-c04c-4158-95af-95f2d60238ef
       }
```

Comment:
think you might have left these by mistake

Reply:
these are just randomly generated uuids, we need them to be uuid so that the jwt encoder does not blow up

Reply:
okay cool